### DISCUSSION TITLE
how can i keep scrolling bottom in RichText?

### DISCUSSION THREAD

**Discussion by tanberBro** [fyne-io/fyne#4466] [Q&A]
when i Refresh(), how can i keep scrolling bottom in RichText?

**Comment by andydotxyz** [comment]
That functionality is not exposed for RichText.
I assume you're making some sort of long appending view - I would recommend against using a single RichText for this, that is not very efficient, as it has to parse NewLines on every refresh to figure out how to lay the text out.

**Reply by tanberBro** [reply]
Which should I use in the long append view?

**Reply by andydotxyz** [reply]
List is set up to handle this (like all the collection widgets it is very efficient - Tree Table and GridWrap too).

If you had multiple re-usable rich texts (or labels) in a list it would be very efficient.
